src/protagonist/API.Tests/Integration/CustomerQueueTests.cs

```diff
@@ -819,6 +825,7 @@ public async Task Post_CreateBatch_201_WithMixtureOfIdSet()
         Guid.TryParse(assetInDatabase.ToList()[0].Id.Asset, out _).Should().BeTrue();
         Guid.TryParse(assetInDatabase.ToList()[1].Id.Asset, out _).Should().BeTrue();
         assetInDatabase.ToList()[2].Id.Asset.Should().Be("someId");
+        assetInDatabase.ToList().All(a => a.Manifests.ContainsOnly("first")).Should().BeTrue();
```

Comment:
As above `Post_CreateBatch_201_WithMixtureOfIdSet` is specifically testing that `"id"` can be empty, missing or set and it's resolved. Assertion on `manifests` doesn't fit the test.